### PR TITLE
:lipstick: create book with core

### PR DESCRIPTION
**Pull request by CDjepeno**

### CODE REVIEW

File: src/infra/Lambda/.txt [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+
+export const getDataSource = (delay = 3000): Promise<DataSource> => {
+  if (AppDataSource.isInitialized) return Promise.resolve(AppDataSource);
+
+  return new Promise((resolve, reject) => {
+    setTimeout(() => {
+      if (AppDataSource.isInitialized) resolve(AppDataSource);
+      else reject("Failed to create connection with database");
+    }, delay);
+  });
```
Comment (CDjepeno) [inline src/infra/Lambda/.txt:138]: A supprimer

File: src/infra/Lambda/package.json [hunk 2]
```diff
@@ -26,9 +26,17 @@
...
     "babel-jest": "^29.1.2",
     "constructs": "^10.0.0",
     "esbuild": "^0.15.10",
+=======
+    "babel-jest": "^29.2.1",
+    "class-validator": "^0.13.2",
+    "constructs": "^10.1.137",
+    "dotenv": "^16.0.3",
+    "esbuild": "^0.15.12",
+>>>>>>> Stashed changes
```
Comment (CDjepeno) [inline src/infra/Lambda/package.json]: A corriger

**Review by CDjepeno — COMMENTED** [review]
In progress...

File: src/infra/Lambda/.txt (hunk 1 above)
Comment (hassanrobleh) [reply src/infra/Lambda/.txt:138]: ok

File: src/infra/Lambda/package.json (hunk 2 above)
Comment (hassanrobleh) [reply src/infra/Lambda/package.json]: C'est corrigé

**Pull request closed without merging**